crates/lsp/src/handlers_state.rs

```diff
@@ -156,6 +157,7 @@ pub(crate) fn did_change(
 pub(crate) fn did_close(
     params: DidCloseTextDocumentParams,
     state: &mut WorldState,
+    auxiliary_event_tx: &AuxiliaryEventSender,
```

Comment:
Any LSP request/notification that needs to log should add a `auxiliary_event_tx` argument that is passed through from the top level `handle_event()`